MAINTAINERS.md

```diff
@@ -0,0 +1,101 @@
+are only able to measure PR contributions. As we improve our project tooling,
+we'll be able to measure and analyze other types of contributions and take those
+into account when evaluating a maintainer application.
+
+### 2. Be experienced in the project
+
+You need to have more than 50 contributions total and project participation for
+more than a year in the Kubernetes organization.
+
+### 3. Find 3 senior maintainer sponsors
+
+As you contribute to Kubernetes, you'll be collaborating with different
+community members. Having the approval of senior project maintainers means that
+they have seen your work and they believe that you'll be a good addition to the
+project. 
+
+**Motivation:** We want the sponsorships to be meaningful, so we require the sponsors to be
+maintainers that collaborated with you and we also don't want to burden the
+senior maintainers otherwise they'll be reluctant to sponsor anyone. Therefore,
+rather than having a mentorship process where a new maintainer is guided by a
+senior member, we want to see the capabilities of the candidate before being
+accepted.
+
+**TODO:** Detail the meaning of a senior project maintainer with a requirement
+along the lines of more than 6 months experience as a project maintainer.
+
+### 4. Apply for maintainership to contributor leads
+
+The application will be discussed by the lead maintainers and put to vote. If
+the vote result is negative, do not be discouraged! You will be given feedback
+so that you can improve yourself and apply again. We want to empower our
+community and cultivate a trust relationship that's why we carefully evaluate
+abilities and experience of each candidate.
+
+**TBD:** As of now Brian Grant is the wg-contribex lead that approves new maintainers.
+
+## What we expect from our maintainers
+
+First and foremost, be a good person, as cheeky as it sounds, you are now a
+member of the Kubernetes project and thus represent the project and your fellow
+developers. We value our community tremendously and we'd like to keep
+cultivating a friendly and collaborative environment for our
+members/contributors/users. For more,
+see
+[Kubernetes Code of Conduct](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/community-expectations.md).
+
+Good technical judgement, and knowledge over one or more components of
+Kubernetes, it might be a project under the Kubernetes or it might be a
+package/component in the main repository such as the Kubernetes documentation.
+We determine the maintainer applicant's judgement via the sponsorship
+requirement. For example, a senior maintainer might decide to sponsor an applicant
+after they witnessed the applicants good technical judgement during a design
+discussion, a feature implementation or by simply witnessing the way they
+interact with other contributors.
+
+## Inactive Maintainers
+
+Maintainers deemed inactive after being inactive for 30 days in the project or
```

Comment:
30 days isn't long enough. That could be a parental leave or even vacation. Probably same as the activity period above, such as 3 months.